WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeActivationBottomSheetFragment.kt

```diff
@@ -29,14 +39,30 @@ class ThemeActivationBottomSheetFragment : BottomSheetDialogFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        // Ensures that bottom sheet always open in expanded state even in landscape mode
-        val behavior = from(requireView().parent as View)
-        behavior.state = STATE_EXPANDED
+        // Ensures that bottom sheet always opens in expanded state even in landscape mode
+        from(requireView().parent as View).state = STATE_EXPANDED
 
         with(ThemeActivationBottomSheetBinding.bind(view)) {
+            viewModel = ViewModelProvider(requireActivity(), viewModelFactory).get(ThemesViewModel::class.java)
+            viewModel.themeToActivate.observe(this@ThemeActivationBottomSheetFragment) {
+                themeNameTv.text = it.name
+                infoTv.text = requireActivity().getString(R.string.theme_bottom_sheet_info, it.name)
+                useThemeLayoutMainTextTv.text =
+                        requireActivity().getString(R.string.theme_bottom_sheet_use_theme_layout_main_text, it.name)
+            }
+
             closeButton.setOnClickListener { dismiss() }
-            useThemeLayoutLl.setOnClickListener { toggleSelection(useThemeCheckIv, keepCurrentCheckIv) }
-            keepCurrentLayoutLl.setOnClickListener { toggleSelection(keepCurrentCheckIv, useThemeCheckIv) }
+
+            useThemeLayoutLl.setOnClickListener {
+                toggleSelection(useThemeCheckIv, keepCurrentCheckIv)
+                viewModel.bottomSheetSelection = USE_THEME_HOMEPAGE
+            }
+
+            keepCurrentLayoutLl.setOnClickListener {
+                toggleSelection(keepCurrentCheckIv, useThemeCheckIv)
+                viewModel.bottomSheetSelection = KEEP_CURRENT_HOMEPAGE
+            }
+
             previewThemeButton.setOnClickListener { /*TODO*/ }
             activateThemeButton.setOnClickListener { /*TODO*/ }
```

Comment:
We may call the `onActivateButtonClicked` of the ViewModel even if the functionality is not ready yet.